src/main/java/hockeyapp/HockeyappBuildAction.java

```diff
@@ -23,11 +21,11 @@ public String getIconFileName() {
         return iconFileName;
     }
 
-   public String getDisplayName() {
-       return displayName;
-   }
+    public String getDisplayName() {
```

Comment:
Extra leading spaces?

Reply:
All other files are indented with 4 spaces per tab, this brings this file into line with that.

Reply:
Ah, kk. Thought it was tab, plus space.
 👍